### PR TITLE
Move KSM alerts from Atlas to Turtles

### PR DESCRIPTION
**Pull request by TheoBrigitte**
Towards: https://github.com/giantswarm/giantswarm/issues/27172

Moving all VerticalPodAutoscaler and KubeStateMetrics related alerts from Atlas to Turtles.

/!\ Does Turtles alerting works ?

### CODE REVIEW

**Comment by TheoBrigitte** [comment]
Updated the alerts to use phoenix routing instead of turtles

**Review by hervenicol — APPROVED** [review]
LGTM

**Comment by TheoBrigitte** [comment]
Holding this until this is resolved https://github.com/giantswarm/giantswarm/issues/27937

File: helm/prometheus-rules/templates/alerting-rules/vpa.all.rules.yml [hunk 1]
```diff
@@ -27,5 +27,5 @@ spec:
         cancel_if_scrape_timeout: "true"
         cancel_if_outside_working_hours: "true"
         severity: page
-        team: atlas
+        team: phoenix
```
Comment (fiunchinho) [inline helm/prometheus-rules/templates/alerting-rules/vpa.all.rules.yml:30]: If we switch to team `phoenix`, I guess we should switch `area` to `kaas` too

File: helm/prometheus-rules/templates/recording-rules/service-level.rules.yml [hunk 2]
```diff
@@ -344,7 +344,7 @@ spec:
     # and summed with 1 so the final result is 0 : no error recorded.
     # If up was unsuccessful, there is an error. Up returns 0, multiplied by -1 and summed
     # with 1 so the final result is 1 : 1 error is recorded .
-    - expr: label_replace(sum((up{app=~'vertical-pod-autoscaler.*'} * -1) + 1) by (cluster_id, cluster_type), "label_application_giantswarm_io_team", "atlas", "", "")
+    - expr: label_replace(sum((up{app=~'vertical-pod-autoscaler.*'} * -1) + 1) by (cluster_id, cluster_type), "label_application_giantswarm_io_team", "phoenix", "", "")
```
Comment (fiunchinho) [inline helm/prometheus-rules/templates/recording-rules/service-level.rules.yml:347]: same here about the `area`

**Review by fiunchinho — CHANGES_REQUESTED** [review]
I'd hold this PR until KSM issues are fixed. It's currently paging a lot and `atlas` has the knowledge about the tool and the work that's currently being done.

File: helm/prometheus-rules/templates/alerting-rules/vpa.all.rules.yml (hunk 1 above)
Comment (TheoBrigitte) [reply helm/prometheus-rules/templates/alerting-rules/vpa.all.rules.yml:30]: thanks, fixed

File: helm/prometheus-rules/templates/recording-rules/service-level.rules.yml (hunk 2 above)
Comment (TheoBrigitte) [reply helm/prometheus-rules/templates/recording-rules/service-level.rules.yml:347]: thanks, fixed

**Comment by TheoBrigitte** [comment]
> I'd hold this PR until KSM issues are fixed. It's currently paging a lot and `atlas` has the knowledge about the tool and the work that's currently being done.

Yes we agreed to keep this open for another week, so we can then check if the situation improved with the `KubeStateMetricsDown` alert. (fixes are in place https://github.com/giantswarm/giantswarm/issues/27937#issuecomment-1713944206)

**Comment by Gacko** [comment]
What's the current status on this?

**Comment by TheoBrigitte** [comment]
Is this still needed ? I updated this PR in case we need to merge it.

**Pull request closed without merging**